content/_index.md

```diff
@@ -24,18 +24,29 @@ In short, it would let anyone easily answer the question, "does this product con
 {{< /home/section >}}
 
 {{< home/section title="How?" >}}
+##### How does [GitBOM](glossary/gitbom/) improve software identification and vulnerability management?
 
-With GitBOM, we propose a solution to both the completeness and the efficiency challenges facing other supply chain tools.
+[GitBOM](glossary/gitbom/) proposes a solution to both the completeness and the efficiency challenges facing other supply chain tools.
```

Comment:
Another spot where this PR reverts my change. Was this intentional?

I thought the wording was stronger with a personal/active "We propose" statement rather than an impersonal/passive statement.

Reply:
As a matter of personal preference, I tend to prefer the passive voice of this sort to the active of an unattached we... but that may be a perversity I picked up writing in sciences...

Reply:
Yes, intentional. IMO the front page positions GitBOMs better when stating what GitBOM is, why and how it works without getting into what "we propose". The more informal use of "we" feels to me to be best positioned at the bottom of the page when trying to personalize the message in order to pull others into the community.

Reply:
fair enough. I'll reserve the informal 'we' for the white paper then :)